safe_transaction_service/history/indexers/ethereum_indexer.py

```diff
@@ -409,7 +409,8 @@ def start(self) -> int:
             current_block_number,
         )
         number_processed_elements = 0
-
+        start_block = 0
```

Comment:
I think this values should be `None` and be typed as `start_block: Optional[int] = None` so it's clear they are not assigned and you plan to do it later. Also a comment to say what you will be doing will be great